index.html

```diff
@@ -189,46 +189,130 @@
         Installable web applications
       </h2>
       <p>
-        This document attempts to address the <cite><a href=
-        "https://w3c-webmob.github.io/installable-webapps/">Use Cases and
-        Requirements for Installable Web Apps</a></cite> .
-      </p>
-      <p>
-        A web application is <dfn data-lt="installing">installed</dfn> if the
-        user agent has provided the end-user with a means of instantiating a
-        new <a>top-level browsing context</a> that has the manifest's members
-        applied to it. That is, the manifest's members, or their defaults, are
-        in effect on the <a>top-level browsing context</a> (see <a>application
-        context</a>).
+        A common use of a manifest is for a user agent to <dfn data-lt=
```

Comment:
use case?

a common use-case of a manifest is for a UA to allow installing the web application